Handbook.md

```diff
@@ -38,9 +37,27 @@ __Links:__
 __Links:__
 ### 3.1 Topics introduction
 ### 3.2 Important guidelines
-(Write in bullet points)
+* Easy team integration
+* Increased code quality efficiency and easy for maintaining
+* Reduce code complexity
+* Reduce development cost
+
+* Naming Convention
+* File and folder Naming and Organization
+* Formatiting and Indentation
+* Commenting and Documenting
+* Classes and Functions
+* Testing
 ### 3.3 Topics
-(Note that it's just as important to highlight bad practices to avoid as good practices to follow. Each section should also contain links to the reference articles for further reading.)
+The developer's world has too much coding standards. It is important in a software development to select the better one. This coding standards depends on the language used i.e., in C++ you will not use the same coding standard than if the language will be in Java or in PHP. It will also depend on the company. You will have three best options, the easiest option is to follow a coding standard which already exists, another way will be to follow a coding standard and make small change because it does not feat with your company rules. The last option will be to create your own coding standards.
+
+The most difficults for a developeur is to follow or work with another developer who does not use the same coding standards. During the development you must create new folders and files. It is important to use the same named and structured format to lose less time as possible when you are looking for a specific file. i.e an asset like a picture or a font.
```

Comment:
typo
developeur -> developer

I also took time to change a sentence that didn't sound right IMO

```suggestion
The most difficults for a developer is to follow or work with another developer who does not use the same coding standards. During the development you must create new folders and files. It is important to use the same naming convention and folder structure to save as much time as possible when you are looking for a specific file. i.e an asset like a picture or a font.
```